meta/main.yml

```diff
@@ -43,15 +43,36 @@ galaxy_info:
     - cd
 
 dependencies:
+
   - name: williamyeh.oracle-java
     # https://github.com/William-Yeh/ansible-oracle-java/pull/60
     src: https://github.com/gjedeer/ansible-oracle-java
     version: "0463628"
     java_remove_download: false
+
   - src: geerlingguy.jenkins
     # use williamyeh.oracle-java instead
     # https://github.com/geerlingguy/ansible-role-java/issues/42
     java_packages: []
     # handle the plugins ourselves - see `jenkins_plugin` use in tasks
     jenkins_plugins: []
-  - src: geerlingguy.repo-epel
+
+  - role: geerlingguy.repo-epel
+
+  - role: geerlingguy.nginx
+    nginx_remove_default_vhost: true
+    nginx_server_tokens: "off"
+    # use EPEL version
+    nginx_yum_repo_enabled: false
+    # https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+behind+an+NGinX+reverse+proxy#JenkinsbehindanNGinXreverseproxy-RunningfromasubdomainwithSSL
+    nginx_upstreams:
+      - name: jenkins
+        servers:
+          - "{{ jenkins_hostname }}:{{ jenkins_http_port }} fail_timeout=0"
```

Comment:
Hmm, seems we need to use upstreams to get the various [configuration options](https://nginx.org/en/docs/http/ngx_http_upstream_module.html#directives), which we are using here, and seem like others could add value. Going to leave it here for now, though we can certainly revisit later.